src/Admin/Controls/ProductGridFactory.php

```diff
@@ -114,6 +114,33 @@ public function create(array $configuration): Datagrid
 			//return $product->primaryCategory->name;
 			return \implode('&nbsp;|&nbsp;', $product->categories->toArrayOf('name'));
 		});
+
+		$grid->addColumn('Obsah', function (Product $object, Datagrid $datagrid) {
+			if ($object->supplierContentLock) {
+				if ($object->content) {
```

Comment:
Tady to bych udělal spíše ternárním operátor / nebo kontatenací bude to přehlednější